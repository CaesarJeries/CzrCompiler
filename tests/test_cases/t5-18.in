int foo(
    int a0,
    int a1,
    int a2,
    int a3,
    int a4,
    int a5,
    int a6,
    int a7,
    int a8,
    int a9
) {
    print("a6=");
    printi(a6);
    print("\na9=");
    printi(a9);
    print("\n");
	return 0;
}

int bar(int x, int y)
{
    return y-x;
}

int fu(int x) {
    return 10-x;
}


void main() {
    foo(0,1,2,3,4,5,6,7,8,9);
    foo(0,1+bar(4,19),2*19,3,4,5,6+bar(2,3)/fu(0),7,8,9-fu(20));
}