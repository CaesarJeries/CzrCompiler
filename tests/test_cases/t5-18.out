Loaded: ./exceptions.s
a6=6
a9=9
a6=6
a9=19
